dev_guide/pod_autoscaling.adoc

```diff
@@ -212,21 +213,148 @@ $ oc create -f hpa.yaml
 
 To view the status of a horizontal pod autoscaler:
 
-====
+* Use the `oc get` command to view information on the CPU utilization and pod limits:
++
 ----
-$ oc get hpa/frontend
-NAME              REFERENCE                                 TARGET    CURRENT   MINPODS        MAXPODS   AGE
-frontend          DeploymentConfig/default/frontend/scale   80%       79%       1              10        8d
+$ oc get hpa/hpa-resource-metrics-cpu
+NAME                         REFERENCE                                 TARGET    CURRENT  MINPODS        MAXPODS    AGE
+hpa-resource-metrics-cpu     DeploymentConfig/default/frontend/scale   80%       79%      1              10         8d
+----
++
+The output includes the following:
++
+** *Target*. The targeted average CPU utilization across all pods controlled by the deployment configuration.
+** *Current*. The current CPU utilization across all pods controlled by the deployment configuration.
+** *Minpods/Maxpods*. The minimum and maximum number of replicas that can be set by the autoscaler.
 
-$ oc describe hpa/frontend
-Name:                           frontend
+* Use the `oc describe` command for detailed information on the horizontal pod autoscaler object.
++
+----
+$ oc describe hpa/hpa-resource-metrics-cpu
+Name:                           hpa-resource-metrics-cpu
 Namespace:                      default
 Labels:                         <none>
 CreationTimestamp:              Mon, 26 Oct 2015 21:13:47 -0400
 Reference:                      DeploymentConfig/default/frontend/scale
-Target CPU utilization:         80%
-Current CPU utilization:        79%
-Min pods:                       1
-Max pods:                       10
+Target CPU utilization:         80% <1>
+Current CPU utilization:        79% <2>
+Min replicas:                   1 <3>
+Max replicas:                   4 <4>
+ReplicationController pods:     1 current / 1 desired
+Conditions: <4>
+  Type                  Status  Reason                  Message
+  ----                  ------  ------                  -------
+  AbleToScale           True    ReadyForNewScale        the last scale time was sufficiently old as to warrant a new scale
+  ScalingActive         True    ValidMetricFound        the HPA was able to successfully calculate a replica count from pods metric http_requests
+  ScalingLimited        False   DesiredWithinRange      the desired replica count is within the acceptable range
+Events:
+----
++
+<1> The average percentage of the requested memory that each pod should be using.
+<2> The current CPU utilization across all pods controlled by the deployment configuration.
+<3> The minimum number of replicas to which to scale down.
+<4> The maximum number of replicas to which to scale up.
```

Comment:
I'm not sure if the above two lines are accurately describing the values. Maybe "The minimum number of replicas to scale down to" and "The maximum number of replicas to scale up to". Maybe?